### DISCUSSION TITLE
Repo API url for gist

### DISCUSSION THREAD

**Discussion by suntong** [community/community#24645] [Apps, API and Webhooks]
<p>GitHub provides API that represents repo’s latest stage via:</p>
<p><code>https://api.github.com/repos/$USER/$REPO/git/refs/heads/$BRANCH</code><br>
like <a href="https://api.github.com/repos/suntong/ffcvt/git/refs/heads/master" rel="noopener nofollow ugc">https://api.github.com/repos/suntong/ffcvt/git/refs/heads/master</a></p>
<p>Is such API available for gist as well?</p>
<p>For e.g., my gist</p>
<pre><code>"url": "https://api.github.com/gists/ff572f25332dbcf41e922ad632259a08",
"gists_url": "https://api.github.com/users/suntong/gists{/gist_id}",
</code></pre>
<p>would such <em>“gists_url”</em> works? I tried it but got <code>message": "Not Found"</code></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>there is the Gist API</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/rest/reference/gists">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/rest/reference/gists" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181085817-85ac723c-571a-45bc-8c4a-459df89429b0.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/rest/reference/gists" target="_blank" rel="noopener">Gists - GitHub Docs</a></h3>

  <p>The Gists API enables the authorized user to list, create, update and delete the public gists on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>there is</p>
<p><a href="https://docs.github.com/en/rest/reference/gists#list-gist-commits">https://docs.github.com/en/rest/reference/gists#list-gist-commits</a></p>
<p>and</p>
<p><a href="https://docs.github.com/en/rest/reference/gists#get-a-gist-revision">https://docs.github.com/en/rest/reference/gists#get-a-gist-revision</a></p>
<p>you may have a workaround that fits your need</p>
